adapters/wazirx-nft-Xsale-bsc/index.js

```diff
@@ -0,0 +1,131 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const Ethereum = require("../../sdk/EVMC");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+const { AVAILABLE_PROTOCOLS } = require("../../sdk/constants");
+
+const WRX_TOKEN = "0x8e17ed70334c87ece574c9d537bc153d8609e2a3";
+const BNB_TOKEN = "bnb";
+
+class WazirxNft {
+    constructor() {
+        this.name = "WazirXNFT";
+        this.symbol = "WRXNFT";
+        this.currencyCode = {
+            "0": {
+                symbol: "WRX",
+                decimals: 8,
+                token_address: WRX_TOKEN,
+            },
+            "1": {
+                symbol: "WRX",
+                decimals: 8,
+                token_address: WRX_TOKEN,
+            },
+            "2": {
+                symbol: "BNB",
+                decimals: 18,
+                token_address: BNB_TOKEN,
+            },
+        };
+        this.token = BNB_TOKEN;
+        this.protocol = "bsc";
+        this.block = 14571030;
+        this.contract = "0x0edabdb72be02cc7cf1d29894aca1b1053286919";
+        this.events = ["TokenSold"];
+        this.pathToAbi = path.join(__dirname, "./abi.json");
+        this.range = 500;
+        this.chunkSize = 6;
+        this.sdk = new Ethereum(this);
+    }
+
+    run = async () => {
+        const s = await this.getSymbol();
+        this.sdk = this.loadSdk();
+        this.symbol = s;
+        await this.sdk.run();
+    };
+
+    loadSdk = () => {
+        return new Ethereum(this);
```

Comment:
it's the wrong sdk

Reply:
I did not find any SDK for BSC. I have renamed Ethereum to BSC by following the bazaar adapter